app/views/content/salaries-and-benefits.md

```diff
@@ -56,57 +56,52 @@ keywords:
 
 $new-salaries$
 
-For helping to shape the next generation, you’re entitled to a competitive salary, generous holidays, and a substantial pension.
+If you’re a primary or secondary teacher in England, your salary will depend on the type of school you work in, where, and the pay range you’re on. 
 
-These salaries apply to maintained primary, secondary and special schools in England (schools funded by local authorities).
+These pay ranges apply to maintained schools (schools funded by local authorities). 
 
-Other types of schools (for example, academies) can set their own pay scales.
+Non-maintained schools (for example, academies, independent schools and private schools) can set their own pay. 
 
-Figures apply from September 2023.
+Figures apply from 1 September 2023.
 
-## Primary and secondary teaching salaries
+## Primary and secondary teacher salary
 
-All qualified primary and secondary teachers will have a starting salary of at least £30,000. This will be higher for teachers working in London. 
+You’ll start on at least £30,000 (or more in London) as a primary or secondary school teacher in England. 
 
-### Qualified teacher salary
+Your salary will be reviewed every year, with most teachers moving up the pay range annually (this will depend on your school’s performance management arrangements).    
 
-Your school will have their own pay scales for qualified teachers. Pay increases will always be linked to performance, not length of service, and will be reviewed every year.
+A typical teacher with 5 years’ experience will earn at least £41,333 (or more in London).
 
-The teacher pay scales for qualified teachers are:
+### Qualified teacher salary
 
 | Area                                     | Minimum | Maximum |
 | -------                                  | -----   | -----   |
-| England (excluding London)     | £30,000 | £41,333 |
-| London fringe                            | £31,350 | £42,689 |
-| Outer London                             | £34,514 | £46,001 |
-| Inner London                             | £36,745 | £47,666 |
-
-Most established teachers will earn more than this maximum by progressing onto the upper pay range for teachers, or by becoming a leading practitioner.
+| England (excluding London)     | £30,000 | £46,525 |
+| London fringe                            | £31,350 | £47,839 |
+| Outer London                             | £34,514 | £51,179 |
+| Inner London                             | £36,745 | £56,959 |
 
-If you’re not sure whether you'll be teaching in inner, outer or the fringe of London, talk to your school.
+### Additional payments
```

Comment:
@gemmadallmandfe @philspray I've not included the actual TLR/SEN pay tables here and would be interested in your thoughts on this! I thought it might be enough to highlight the fact that they could be eligible for additional payments without going into the specifics - feel like if we started talking about TLRs 1,2 and 3, we'd have to explain the differences between them which could get more complicated. But equally happy to add them back in if you think worth it!

Reply:
@EllieNodder I'm happy with this approach if the pay team are - as long as they are comfortable that we are not being misleading by only quoting the highest figure